tests/python/gaia-ui-tests/gaiatest/apps/system/regions/cards_view.py

```diff
@@ -14,7 +14,7 @@
 class CardsView(Base):
 
     # Home/Cards view locators
-    _cards_view_locator = (By.ID, 'cards-view')
+    _task_manager_locator = (By.ID, 'task-manager')
```

Comment:
Yes, I agree. I noticed lots of these regions do need to be converted to use PageRegion like that, right?